src/app/markdown/markdown.component.scss

```diff
@@ -0,0 +1,70 @@
+@import "../../assets/stylesheets/_base";
```

Comment:
Please remove the extra lines between each class. The line after an ending `}` should be the next class.